resources/js/Store/cart.js

```diff
@@ -0,0 +1,51 @@
+import { createStore } from 'vuex';
+import createPersistedState from 'vuex-persistedstate'
+
+export default createStore({
+  plugins: [createPersistedState({
+      storage: window.sessionStorage,
+  })],
+  state: {
+    items: []
+  },
```

Comment:
For all intents and purposes of this exam, the use of a persistent `vuex` shows resourcefulness and is commendable. But in actual production environments, we shall recommend that the cart be also stored in the backend database for better persistence.

As it stands, when this user logs out, we also clear the sessionStorage and so their cart is lost. Having it in the backend will help us pull their cart data even if they log on a different device.